modules/android/vsandroid_vcxproj.lua

```diff
@@ -16,6 +16,25 @@
 	local config = p.config
 
 
+--
+-- Utility functions
+--
+
+	function table.findAndRemove(t, value)
```

Comment:
This appears to be unused and can probably be removed.

Reply:
Removed the function for this PR, but as @WorldofBay said, I hope we can get it in `table.lua` someday.